store/src/main/java/com/book/store/model/Order.java

```diff
@@ -0,0 +1,50 @@
+package com.book.store.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @NotNull
+    private User user;
+
+    @Enumerated(EnumType.STRING)
+    @NotNull
+    private Status status;
+
+    @NotNull
+    private BigDecimal total;
+
+    @NotNull
+    private LocalDateTime orderDate;
+
+    @NotBlank
+    private String shippingAddress;
```

Comment:
There are validation annotations, you need to add database constraints, e.g. Using @Column
```suggestion
    private User user;

    @Enumerated(EnumType.STRING)
    private Status status;

    private BigDecimal total;

    @NotNull
    private LocalDateTime orderDate;


    private String shippingAddress;
```